### DISCUSSION TITLE
Re initialise the app after changing firebase project

### DISCUSSION THREAD

**Discussion by men3malgmatti** [invertase/react-native-firebase#5492] [Q&A] [answered]
Hello I have question , i followed the installation steps for android and after connecting to my firebase project successfully, i want to connect to a new firebase project replacing the old one, i changed the google-services.json file, but i am still connected to the old firebase project, how can I re-initialise the default app with new firebase project ?

**Comment by mikehardy** [comment, marked as answer]
This is too vague to offer advice sorry, it could be any of a number of things. You'll need the ios plist file as well probably? Maybe `npx react-native-clean-project`

**Reply by men3malgmatti** [reply]
Thanks a lot for your reply , i had to do a work around because i tried all solutions and they didn't work for me
